etc/nginx/proxy.conf

```diff
@@ -1,8 +1,23 @@
+# Global settings
+# Send access and error logs to container stdout
+# So they automatically show up in "oc logs" output / Splunk
+access_log /dev/stdout combined;
+error_log /dev/stdout;
```

Comment:
I especially like this, thank you! Bonus points if we could add `app=corgi` and `environ={stage/prod}` to the message by default for filtering in Splunk.